worker.go

```diff
@@ -175,8 +157,12 @@ func buildManagedModel(certsProvider certs.CertificatesProvider, ingresses []*ku
 		}
 		if item, ok := model[certificateARN+"/"+ingress.Scheme()]; ok {
 			item.ingresses = append(item.ingresses, ingress)
+			item.Update = true
```

Comment:
This Update = true is in both cases set. We have to lookup it somewhere to decide if there was a change that we now want to react on. Maybe we have a currentKnownState and check the new data to this one, but this would mean we are becoming stateful and are not stateless anymore. I don't see another way of doing it or trigger updates by time.Duration and additionally in the initial run.

Reply:
The state is in the cloudformation stack as it has the certificate arn and scheme defined. We just need to check against that an decide if there was a change.